README.md

```diff
@@ -57,14 +57,25 @@ problems. Community support is also available in the
 LND and the loop client are using Go modules. Make sure that the `GO111MODULE`
 env variable is set to `on`.
 
-In order to execute a swap, **You need to run lnd 0.7.1+, or master built with
-sub-servers enabled.**
+In order to execute a swap, **you need to run a compatible lnd version built
+with sub-servers enabled.**
 
 ### LND
 
-If you are building from source, and not using a 0.7.1 or higher release of
-lnd, make sure that you are using the `master` branch of lnd. You can get this
-by git cloning the repository
+To run loop, you need a compatible version of `lnd` running. It is generally
+recommended to always keep both `lnd` and `loop` updated to the most recent
+released version. If you need to run an older version of `lnd`, please consult
+the following table for supported versions.
+
+LND Version | Compatible Loop Version
+------------|------------------------
+`v0.10.x-beta`                 | `>= v0.6.0-beta`
+`v0.7.1-beta` - `v0.9.2-beta` | `v0.5.1-beta`
+`<= v0.7.0-beta`              | not supported for loop!
+
+
+If you are building from source make sure that you are using the `master` branch
+of lnd. You can get this by git cloning the repository:
```

Comment:
I know this was already in the readme, but do we want to be telling people to run lnd on master? Wouldn't we want them to build the most recent tag?

Reply:
You are right. I don't think we should advise people to run master. I changed the text.